rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/facade/copy.kt

```diff
@@ -106,12 +106,12 @@ fun String.copyId(): CopyId {
     return when {
         this.contains('/') -> {
             this.split("/,").let {
```

Comment:
This split is wrong. I have noted it right now during code review. The correct split should be: `this.split("/", ",")`.